src/csharp/Grpc.Core.Api/Grpc.Core.Api.csproj

```diff
@@ -31,4 +32,8 @@
     <Reference Include="Microsoft.CSharp" />
   </ItemGroup>
 
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' ">
+    <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="1.0.0-preview7.19362.9" />
```

Comment:
looks like this package requires net461 so it's not going to work with net45 target.